aten/src/ATen/native/native_functions.yaml

```diff
@@ -6725,6 +6725,28 @@
   variants: function, method
   device_guard: False
 
+- func: isposinf(Tensor self) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+
+- func: isposinf_(Tensor(a!) self) -> Tensor(a!)
```

Comment:
Let's not implement an in-place variant for these functions right now. It would only work on bool tensors, anyway.

Reply:
The in-place variants are deleted in the latest commit.